frontend/src/metabase/query_builder/containers/test-utils.tsx

```diff
@@ -379,8 +381,15 @@ export const waitForSaveToBeEnabled = async () => {
   });
 };
 
-export const waitForNativeQueryEditoReady = async () => {
+export const waitForNativeQueryEditorReady = async () => {
   await waitFor(() => {
     expect(screen.getByTestId("mock-native-query-editor")).toBeInTheDocument();
   });
 };
+
+export const waitForFaviconReady = async (container: HTMLElement) => {
```

Comment:
`QueryBuilder.unit.spec.tsx` was flaking with very high probability without this.